plugins/catalog-backend-module-incremental-ingestion/src/module/WrapperProviders.ts

```diff
@@ -75,14 +75,14 @@ export class WrapperProviders {
   }
 
   async adminRouter(): Promise<express.Router> {
-    return createIncrementalProviderRouter(
+    return await new IncrementalProviderRouter(
       new IncrementalIngestionDatabaseManager({ client: this.options.client }),
       loggerToWinstonLogger(this.options.logger),
-    );
+    ).createRouter();
   }
 
-  private async startProvider(
-    provider: IncrementalEntityProvider<unknown, unknown>,
+  private async startProvider<TCursor, TContext>(
+    provider: IncrementalEntityProvider<TCursor, TContext>,
```

Comment:
could revert this change now as well, right? Same "T vs unknown" situation as the other one

Reply:
Done!